xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd

```diff
@@ -256,7 +256,7 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Date of last trip.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="CommentsAbout" type="MultilingualString" minOccurs="0">
+			<xsd:element name="CommentsAbout" type="MultilingualString" minOccurs="0" maxOccurs="unbounded">
```

Comment:
This is a functional change. Where does it come from?

Reply:
It is. but I think it was wrong in the beginning. CommentsAbout as a single MultilingualString seems not modeled right. Remove and a separate functional change?

Reply:
Yes, remove, and for next we introduce it. I would even want to have a discussion with @Aurige regarding "Name" specifically for translations.

Reply:
And **AlternativeName** when it is specific for names